Tasks/priority_queue.py

```diff
@@ -0,0 +1,69 @@
+"""
+Priority Queue
+
+Queue priorities are from 0 to 5
+"""
+
+my_pri_qu = [[] for i in range(6)]
+
+def enqueue(elem, priority: int = 0) -> None:
+	"""
+	Operation that add element to the end of the queue
+
+	:param elem: element to be added
+	:return: Nothing
+	"""
+	if priority == 0:
```

Comment:
А что вам мешало написать...
```suggestion
	my_pri_qu[priority].append(elem)
```
 ? =)